### PR TITLE
(maint) update changelog for 0.4.7 release.

### PR DESCRIPTION
**Pull request by adreyer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
+## 0.4.7
```
Comment (cprice404) [inline CHANGELOG.md]: Should probably be 0.5.0?  Includes new API?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+ This is a feature release.
+
+* [TK-312] Unbuffered streams in Java to match clojure support released in 0.4.5.
```
Comment (cprice404) [inline CHANGELOG.md]: Link to Jira ticket?

File: CHANGELOG.md (hunk 1 above)
Comment (adreyer) [inline CHANGELOG.md]: That seems better. It wasn't clear from the previous releases when we bumped what.

Comment (cprice404) [inline CHANGELOG.md]: Yeah I think the release where the buffering went in should have been a Y, but it was accidentally done as a Z.

File: CHANGELOG.md (hunk 2 above)
Comment (adreyer) [inline CHANGELOG.md]: There wasn't one specifically should there be a post-facto ticket?

Comment (cprice404) [inline CHANGELOG.md]: for TK-312?

Comment (cprice404) [inline CHANGELOG.md]: You don't need to add a ticket for the other one, just saying we should add the link for 312.

**Pull request merged**